src/PickerInput/hooks/useRangeValue.ts

```diff
@@ -289,11 +289,7 @@ export default function useRangeValue<ValueType extends DateType[], DateType ext
 
       // Trigger `onChange` if needed
       if (onChange && !isSameMergedDates) {
-        onChange(
-          // Return null directly if all date are empty
```

Comment:
It was difficult for me to understand why such specifications were made 😢 
I appreciate if someone tells me. If my change is not good for that, I also want to find better solution!